### PR TITLE
Add information about JAliEn and alien.py

### PR DESCRIPTION
**Pull request by Atlantic777**
Adding JAliEn and alien.py related updates. Mostly notes that alien-token-init is now optional, and switch from aliensh to alien.py.

### CODE REVIEW

File: analysis/setup.md [hunk 1]
```diff
@@ -134,6 +177,7 @@ Move to the directory where the data we are looking for is stored
 ```
 cd /alice/data/2015/LHC15o/000246757/pass1/AOD/002/
 ```
+NOTE: this example needs to be updated
```
Comment (mpuccio) [inline analysis/setup.md]: What needs to be updated specifically? Is it just the syntax for alien.py?

Comment (Atlantic777) [reply analysis/setup.md]: That was note for myself, I should remove it. The alien.py examples are already added.

**Comment by Atlantic777** [comment]
Ciao Max, yes, let's add a link to [JAliEn docs](https://jalien.docs.cern.ch). Perhaps [here](https://alice-doc.github.io/alice-analysis-tutorial/analysis/setup.html#get-a-valid-token)?

**Comment by mpuccio** [comment]
Yes! I think that there is perfect! The PR is fine with me. As soon as the tests are done I will merge it.
Thanks a lot!

**Comment by Atlantic777** [comment]
I rebased the branch but now it must be approved again.

**Pull request merged**